src/printer.cpp

```diff
@@ -36,6 +37,9 @@ limitations under the License.
 
 namespace libcellml {
 
+static const std::regex before(">[\\s\n\t]*");
+static const std::regex after("[\\s\n\t]*<");
```

Comment:
No need for it to be outside of `printMath()` (unless we were ever to use those regular expressions, but we are not for now, so...

Reply:
Ah, I thought that's what you wanted!  OK, will move it back.

Reply:
cc6a4619